### DISCUSSION TITLE
Make clear what a new release offers

### DISCUSSION THREAD

**Discussion by 1n3JgKl9pQ6cUMrW** [admob-plus/admob-plus#276] [Ideas]
### Is this a bug report?

**No.**

---

I am following this excellent plug-in for months now, but there is one thing that is a bit annoying;

It has "too many" updates following after each other.

I am using version 1.0.0 for about one week one (used 0.37 during bèta-testing of my own apps) and one week later we are already at 1.3.0

It's not the updates that are the issue, but the lack of information about every new version (sometimes there are two new versions a day).

Of course I admire the hard work, but maybe it's possible to publish an update-log or something?

Like with "real" apps?

Just show two or three highlights for every release, so one does not have to read the source-code or check every file-diff to see what has changed.

For most of us it's not always a good idea to update the plug-in with every release, because it can cause issues for the app itself (or production flow).

---

### November 21st, 2020

1. Fixed 'banner top' position
2. Added new debug option

#### August 27th, 2020

1. Added extra CLI options
2. Fixed "interstitial not closing" issue
3. Added reward video model

---

...something like that? It can be published on the site, so one can keep track on it.

**Comment by ratson** [comment]
@1n3JgKl9pQ6cUMrW Thanks for bringing this up.
I know it is useful to have detailed release notes, but an automatic generated CHANGELOG is the best I could offer
https://github.com/admob-plus/admob-plus/blob/master/packages/cordova/CHANGELOG.md

Do you want to help  rewriting them into a separate, more readable release note?

**Reply by 1n3JgKl9pQ6cUMrW** [reply]
To be honest I did not even realize that file was there.

It's almost the changelog I described in my proposal - so not much has to be done here.

Maybe you can link it directly (or integrate) on the website?

I use the site as a reference (for the API / events / options) and Git only for downloading the package (using the CLI, so I never see it in a browser).

When there's a changelog on the site as well, things become much clearer :-)

**Reply by ratson** [reply]
@1n3JgKl9pQ6cUMrW There is already links to CHANGELOG on the website.
![changelog-link](https://user-images.githubusercontent.com/2682937/112867314-5a34f100-90ed-11eb-9543-689db361b80d.png)
